src/renderer/components/tooltip/view.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+class Tooltip extends React.PureComponent {
+  render() {
+    const { text, placement = 'top', children } = this.props
```

Comment:
Can you add the `placement: top` value to `defaultProps` ?

Reply:
Added. Can you check it?